server/market/market.go

```diff
@@ -587,6 +588,163 @@ func (m *Market) SubmitOrder(rec *orderRecord) error {
 	return <-m.SubmitOrderAsync(rec)
 }
 
+// processCancelOrderWhileSuspended is called when cancelling an order while
+// the market is suspended and Run is not running. The error sent on errChan
+// is returned to the client.
+//
+// This function:
+// 1. Removes the target order from the book.
+// 2. Unlocks the order coins.
+// 3. Updates the storage with the new cancel order and cancels the existing limit order.
+// 3. Responds to the client that the order was received.
+// 4. Sends the unbooked order to the order feeds.
+// 5. Creates a match object, stores it, and notifies the client of the match.
+func (m *Market) processCancelOrderWhileSuspended(rec *orderRecord, errChan chan<- error) {
+	co, ok := rec.order.(*order.CancelOrder)
+	if !ok {
+		errChan <- ErrInvalidOrder
+		return
+	}
+
+	if cancelable, err := m.CancelableBy(co.TargetOrderID, co.AccountID); !cancelable {
+		errChan <- err
+		return
+	}
+
+	m.bookMtx.Lock()
+	delete(m.settling, co.TargetOrderID)
+	lo, ok := m.book.Remove(co.TargetOrderID)
+	m.bookMtx.Unlock()
+	if !ok {
+		errChan <- ErrTargetNotCancelable
+		return
+	}
+
+	m.unlockOrderCoins(lo)
+
+	sTime := time.Now().Truncate(time.Millisecond).UTC()
+	co.SetTime(sTime)
+
+	// Create the client response here, but don't send it until the order has been
+	// committed to the storage.
+	respMsg, err := m.orderResponse(rec)
+	if err != nil {
+		errChan <- fmt.Errorf("Failed to create order response: %v", err)
```

Comment:
%w and lower case `error` message: `fmt.Errorf("failed to create order response: %w", err)`

Reply:
Nice, I didn't know about these wrapping errors.